build.gradle

```diff
@@ -36,7 +39,8 @@ version = project.hasProperty('project.version') ? project.getProperty('project.
 sourceSets.main.java.srcDirs = []
 sourceSets.main.groovy.srcDirs += ["src/main/java"]
 
-sourceCompatibility = '1.8'
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
 
 configurations {

```

Comment:
Really is needed to hardcode all versions defined in spring boot? This is nightmare when spring boot is updated.